plugins/glossary-transformer.js

```diff
@@ -50,8 +50,23 @@ function createGlossaryTransformer(options = {}) {
   }
   
   function shouldProcess(filePath, fileContent) {
-    return filePath?.endsWith('.mdx') && 
+    const allowedPaths = [
+      'docs/concepts/*',
+      'docs/managing-data/core-concepts/*',
+      'docs/getting-started/*'
+    ];
```

Comment:
We seem to be processing all the files when only a small portion have `^^ ^^` so let's just do a subset. We can add as we go along.